src/main/java/io/confluent/connect/elasticsearch/Validator.java

```diff
@@ -100,25 +100,27 @@ public Config validate() {
       return new Config(validations);
     }
 
-    try (RestHighLevelClient client = clientFactory.client()) {
-      validateCredentials();
-      validateDataStreamConfigs();
-      validateIgnoreConfigs();
-      validateKerberos();
-      validateLingerMs();
-      validateMaxBufferedRecords();
-      validateProxy();
-      validateSsl();
-
-      if (!hasErrors()) {
-        // no point if previous configs are invalid
+    validateCredentials();
+    validateDataStreamConfigs();
+    validateIgnoreConfigs();
+    validateKerberos();
+    validateLingerMs();
+    validateMaxBufferedRecords();
+    validateProxy();
+    validateSsl();
```

Comment:
Any of the validations moved out of try catch don't throw an exception?

Reply:
None from the looks of things. Also earlier, only IO exception related to closing client was handled so I presume these ones don't have any scope to throw any exception.

Reply:
ok